src/EditorFeatures/TestUtilities2/Intellisense/Completions.vb

```diff
@@ -0,0 +1,7 @@
+﻿' Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+Namespace Microsoft.CodeAnalysis.Editor.UnitTests.IntelliSense
+    Public Enum Completions
+        OldCompletion = 1
```

Comment:
:bulb: In this branch, suggest the name `Legacy`, and in the **asynccompletion** branch add the value `Modern`